src/content/docs/contributing/nf_core_basic_training.md

```diff
@@ -30,64 +45,72 @@ The course is going to build an (totally unscientific and useless) RNA seq pipel
 
-**2. Exploring the nf-core tools command**
+3. **Creating a new nf-core pipeline from the nf-core template**
 
-A very basic walk-through of what can be done with nf-core tools
+4. **Exploring the nf-core template**
 
-**3. Creating a new nf-core pipeline from the nf-core template**
+  a) The git repository
 
-**4. Exploring the nf-core template**
+  b) running the pipeline
 
-a) The git repository
+  c) linting the pipeline
 
-b) running the pipeline
+  d) walk-through of the template files
 
-c) linting the pipeline
+5. **Building a nf-core pipeline using the template**
 
-d) walk-through of the template files
+  a) Adding a nf-core module to your pipeline
 
-**5. Building a nf-core pipeline using the template**
+  b) Adding a local custom module to your pipeline
 
-a) Adding a nf-core module to your pipeline
+  c) Working with Nextflow schema
 
-b) Adding a local custom module to your pipeline
+  d) Linting your modules
 
-c) Working with Nextflow schema
+## Preparation
 
-d) Linting your modules
+### Prerequisites
 
-## Prerequisites
+- Familiarity with Nextflow syntax and configuration.
 
-## Follow the training videos
+### Follow the training videos
 
 This training can be followed either based on this documentation alone, or via a training video hosted on youtube. You can find the youtube video in the Youtube playlist below:
 
 (no such video yet)
 
-# Using gitpod
+### Gitpod
 
-For this tutorial we are going to use Gitpod, which is best for first-timers as this platform contains all the programs and data required.
-Gitpod will contain a preconfigured Nextflow development environment and has the following requirements:
+For this tutorial we will use Gitpod, which runs in the learners web browser. The Gitpod environment contains a preconfigured Nextflow development environment
+which includes a terminal, file editor, file browser, Nextflow, and nf-core tools. To use Gitpod, you will need:
 
 - A GitHub account
 - Web browser (Google Chrome, Firefox)
 - Internet connection
 
-Simply click the link and log in using your GitHub account to start the tutorial:
+Click the link and log in using your GitHub account to start the tutorial:
 
 <p class="text-center">
   <a href="https://www.gitpod.io/#https://github.com/nf-core/basic_training" class="btn btn-lg btn-success" target="_blank">
     Launch GitPod
   </a>
 </p>
 
-For more information about gitpod, including how to make your own gitpod environement, see the gitpod bytesize talk on youtube (link to the bytesize talk)
+For more information about Gitpod, including how to make your own Gitpod environement, see the Gitpod bytesize talk on youtube (link to the bytesize talk),
+check the [nf-core Gitpod documentation](gitpod/index) or [Gitpod's own documentation](https://www.gitpod.io/docs).
 
-## Explore your Gitpod interface
+<details>
+<summary> Expand this section for instructions to explore your Gitpod environment</summary>
+
```

Comment:
would be good to have some space between this expandable area and the next header. Currently it looks a bit cramped.

Reply:
We should add a margin then to the style sheet. @mashehu which is the correct file to edit if I want to add a margin to the bottom of the details tag?